octoprint_Spoolman/modules/PrinterHandler.py

```diff
@@ -43,6 +43,9 @@ def handlePrintingStatusChange(self, eventType):
         ):
             self.lastPrintOdometer = None
             self.lastPrintOdometerLoad = None
+            
+            self._settings.set([SettingsKeys.DATA_SAVE], None)
```

Comment:
Please move the newly added code here to a separate function, which will encapsulate this feature better.